shell/nuxt.config.js

```diff
@@ -610,7 +612,16 @@ export default function(dir, _appConfig) {
       ]
     },
 
-    typescript: { typeCheck: { eslint: { files: './**/*.{ts,js,vue}' } } },
+    typescript: {
+      typeCheck: {
+        eslint: {
+          exclude: [
+            './pkg/rancher-components/*'
```

Comment:
Why do we exclude this path?

Reply:
I'd like to see rancher components to start making use of rules that make sense for a component framework (e.g. `vue/no-mutating-props`) so we can properly warn about unintended side effects.

From a previous conversation in this PR 

> I don't find the walking behavior to be desirable as I want rancher-components to make use of @vue/standard so we can start focusing on some improvements in our components. Mutating props is one thing that comes to mind as something that doesn't track as a good practice for a component library